src/pages/accounts/resetPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,58 @@
+import { forgot_password } from '../../../utils/constants';
+import { Button, ErrorBox, InputWithLabel } from '../../../components/reusables/Reusable';
+import { useFormik } from 'formik';
+import { useForgotPassword } from './hooks';
+import { toast, ToastContainer } from 'react-toastify';
+
+interface FormValues {
+  email: string;
+}
+
+export default function ForgotPassword() {
+  const { isSent, handleSubmit } = useForgotPassword();
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+    },
+    validate() {
+      const errors: Partial<FormValues> = {};
+      if (formik.touched.email && !formik.values.email) {
+        errors.email = 'Email is Required';
+      } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(formik.values.email)) {
+        errors.email = 'Invalid email address';
+      }
+      return errors;
+    },
+    onSubmit: (values: FormValues) => {
+      handleSubmit(values);
+    },
+  });
+  if (isSent.error !== '') {
+    toast.error(isSent.error);
```

Comment:
```suggestion
  const formik = useFormik({
    initialValues: {
      email: '',
    },
    validate() {
      const errors: Partial<FormValues> = {};
      if (formik.touched.email && !formik.values.email) {
        errors.email = 'Email is Required';
      } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(formik.values.email)) {
        errors.email = 'Invalid email address';
      }
      return errors;
    },
    onSubmit: (values: FormValues) => {
      handleSubmit(values);
    },
  });
  if (isSent.error !== '') {
    toast.error(isSent.error);
```
can you use the validation package like yup